editor/js/Menubar.File.js

```diff
@@ -393,14 +393,11 @@ Menubar.File = function ( editor ) {
 	//
 
 	var link = document.createElement( 'a' );
-	link.style.display = 'none';
-	document.body.appendChild( link ); // Firefox workaround, see #6594
-
 	function save( blob, filename ) {
 
 		link.href = URL.createObjectURL( blob );
 		link.download = filename || 'data.json';
-		link.click();
+		link.dispatchEvent( new MouseEvent('click') );
```

Comment:
Can you please ensure correct code style. Write it like so:
```js
link.dispatchEvent( new MouseEvent( 'click' ) );
```

Reply:
No problem. Done.